fast_calvo_classifier.py

```diff
@@ -65,65 +71,63 @@ class FastCalvoClassifier(RodanTask):
     Entry point
     """
     def run_my_task(self, inputs, settings, outputs):
-        # Inner configuration
-        mode = 'logical'
-
-        # Ports
-        background_model = inputs['Background model'][0]['resource_path']
-        symbol_model = inputs['Symbol model'][0]['resource_path']  
-        staff_model = inputs['Staff-line model'][0]['resource_path']  
-        text_model = inputs['Text model'][0]['resource_path']
-
-        model_paths = [background_model, symbol_model, staff_model, text_model]
-
-        # Set up log file if applicable
-        logging.getLogger().addHadler(
-                logging.StreamHandler()
-        )
-        if len(outputs['Log File']) > 0:
-            logging.getLogger().addHandler(
-                    logging.FileHandler(outputs['Log File'][0]['resource_path'])
-            )
-
-        # Settings        
-        height = settings['Height']
-        width = settings['Width']
-        threshold = settings['Threshold']
-
-        for idx, _ in enumerate(inputs['Image']):
-            # Process
-            image_filepath = inputs['Image'][idx]['resource_path']
-            image = cv2.imread(image_filepath, True)
-
-            analyses = recognition.process_image_msae(image, model_paths, height, width, mode = mode)
-
-            for id_label, _ in enumerate(model_paths):
-                if mode == 'masks':
-                    mask = ((analyses[id_label] > (threshold / 100.0)) * 255).astype('uint8')
-                elif mode == 'logical':
-                    label_range = np.array(id_label, dtype=np.uint8)
-                    mask = cv2.inRange(analyses, label_range, label_range)
- 
-                original_masked = cv2.bitwise_and(image, image, mask = mask)
-                original_masked[mask == 0] = (255, 255, 255)
-
-                # Alpha = 0 when background
-                alpha_channel = np.ones(mask.shape, dtype=mask.dtype) * 255
-                alpha_channel[mask == 0] = 0
-                b_channel, g_channel, r_channel = cv2.split(original_masked)
-                original_masked_alpha = cv2.merge((b_channel, g_channel, r_channel, alpha_channel))
-
-                if id_label == 0:
-                    port = 'Background'
-                elif id_label == 1:
-                    port = 'Music symbol'
-                elif id_label == 2:
-                    port = 'Staff lines'
-                elif id_label == 3:
-                    port = 'Text'
-
-                if port in outputs:
-                    cv2.imwrite(outputs[port][idx]['resource_path']+'.png', original_masked_alpha)
-                    os.rename(outputs[port][idx]['resource_path']+'.png', outputs[port][idx]['resource_path'])
-
-        return True
+        oldouts = sys.stdout, sys.stderr
+        try:
+            rlevel = app.conf.CELERY_REDIRECT_STDOUTS_LEVEL
+            app.log.redirect_stdouts_to_logger(logger, rlevel)
```

Comment:
With this line, are we still seeing the logs in the Celery queue?

Reply:
I'll have to double check that but I believe so since we are using a celery logger still.

Reply:
Messages still end up in the Celery log for the worker.

Reply:
Great, thanks!